doc/tutorials/using_partition_mode_on_nuc.rst

```diff
@@ -225,6 +223,25 @@ Update Ubuntu GRUB to Boot Hypervisor and Load Kernel Image
       The module ``/boot/ACPI_VM1.bin`` is the binary of ACPI tables for pre-launched VM1 the parameter ``ACPI_VM1`` is
       VM1's ACPI tag and should not be modified.
 
+#. Correct example image
+
+   .. code-block:: console
+
+      menuentry 'ACRN hypervisor Logical Partition Scenario' --id ACRN_Logical_Partition --class ubuntu --class gnu-linux --class gnu --class os $menuentry_id_option 'gnulinux-simple-e23c76ae-b06d-4a6e-ad42-46b8eedfd7d3' {
+           recordfail
+           load_video
+           gfxmode $linux_gfx_mode
+           insmod gzio
+           insmod part_gpt
+           insmod ext2
+           search --no-floppy --fs-uuid --set 9bd58889-add7-410c-bdb7-1fbc2af9b0e1
+           echo 'Loading hypervisor logical partition scenario ...'
```

Comment:
```suggestion
           echo 'Loading ACRN Partitioned Scenario ...'
```

Reply:
The updated document version is verified based on releasev_2.6, and the name cannot be changed temporarily, thank you!

Reply:
Got it